packages/web-pkg/src/constants.ts

```diff
@@ -4,6 +4,7 @@ export const EVENT_TROW_CONTEXTMENU = 'contextmenuClicked'
 
 export abstract class ImageDimension {
   static readonly Thumbnail: [number, number] = [36, 36]
+  static readonly Tile: [number, number] = [128, 128]
```

Comment:
The oCIS thumbnail service creates a square image for this resolution and your css then stretches it to 16:9 aspect ratio. That looks off... as a workaround you could use a higher resolution for now, so that the thumbnail service doesn't crop to a square resolution. 1000x1000 should do. The thumbnail service only has some limited hardcoded solutions and picks the one that is closest to the one you requested.

Reply:
Per https://github.com/owncloud/ocis/blob/b24d126b3086351200c382d2dd46dea5bb05a66b/services/thumbnails/pkg/config/defaults/defaultconfig.go#L40 this would mean 1920x1080 I guess? Quite some big files potentially, but we can go with that for the beginning, yes